README.md

```diff
@@ -20,20 +20,22 @@ Jump in: [![Slack Status](http://slack.projecthydra.org/badge.svg)](http://slack
   * [What is Hyrax?](#what-is-hyrax)
     * [Feature list](#feature-list)
   * [Help](#help)
-  * [Getting started](#getting-started)
+  * [Getting Started](#getting-started)
```

Comment:
The general style used in the README for headers is not to capitalize every word.